subaccounts.go

```diff
@@ -274,27 +249,17 @@ func (c *Client) SubaccountContext(ctx context.Context, id int) (subaccount *Sub
 			slist := map[string]Subaccount{}
 			err = json.Unmarshal(body, &slist)
 			if err != nil {
-				return
```

Comment:
That duplicated `if` block has to be a copy/paste mistake of some kind, removed.

Accepting all 2XX status codes sounds like a good change. I'll make that change in this file and add an issue to take care of it in the rest of the files.